tidb-in-kubernetes/expose-servics.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 服务暴露
+category: how-to
+---
+
+# 服务暴露
+
+Kubernetes 集群内访问 TiDB 使用 TiDB service 域名 `<tidbcluster-name>.<namespace>` 即可。若需要集群外访问，则需将 TiDB 服务端口暴露出去。在 tidb-cluster helm chart 中，通过 values 文件中 `tidb.service` 字段配置：
+
+```yaml
+service:
+  type: NodePort
+  # externalTrafficPolicy: Cluster
+  # annotations:
+  # cloud.google.com/load-balancer-type: Internal
+```
+
+## NodePort
+
+在没有 LoadBalancer 时，可选择通过 NodePort 暴露。NodePort 有两种模式：
+
+- `externalTrafficPolicy=Cluster`: 集群所有的机器都会给 TiDB 分配 TCP
+  端口，此为默认值
+- `externalTrafficPolicy=Local`: 只有运行 TiDB 的机器会分配 TCP 端口，用于访问本地的 TiDB 实例
+
+使用 Local 模式时，建议打开 tidb-scheduler 的 `StableScheduling` 特性。tidb-scheduler 会尽可能在升级过程中将新 TiDB 实例调度到原机器，这样集群外客户端不用在 TiDB 重启后更新配置。
```

Comment:
Maybe we should document or reference how to enable `StableScheduling`

Reply:
Do we have a doc which contains all configuration options of chart's values? I think we can add a link to `tidb-scheduler` configuration section.